tools/setupResourceScripts/setup_tdr_resources.py

```diff
@@ -161,15 +161,32 @@ def add_billing_profile_members(clients, profile_id):
     )
 
 
-def dataset_ingest_json(clients, dataset_id, dataset_to_upload):
+def create_ingest_request_csv(table, upload_prefix):
+    return {
+        "csv_skip_leading_rows": 1,  # change to 0 if there is not a header row
+        "format": "csv",
+        "path": f"{upload_prefix}/{table}.csv",
+        "table": table
+    }
```

Comment:
If it's ok to provide `csv_skip_leading_rows` for json too (I assume it would be ignored) you could combine these methods
```suggestion
def create_ingest_request_csv(table, upload_prefix, format):
    return {
        "csv_skip_leading_rows": 1,  # change to 0 if there is not a header row
        "format": format,
        "path": f"{upload_prefix}/{table}.{format}",
        "table": table
    }
```

Reply:
yeah I tested json ingest and it just ignores the csv_skip_leading_rows property